content/id/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -0,0 +1,212 @@
+
+{{< note >}}
+
+Mengetik Secret pada baris perintah dapat menyimpannya dalam riwayat (_history_) dari _shell_ kamu tanpa perlindungan, dan
+Secret tersebut mungkin juga terlihat oleh pengguna lain dalam PC kamu selama perintah `kubectl` sedang berjalan.
+{{< /note >}}
+
+
+## Menginspeksi Secret `regcred`
+
+Untuk memahami isi Secret `regcred` yang baru saja kamu buat, mulailah dengan melihat Secret dalam format YAML:
+
+```shell
+kubectl get secret regcred --output=yaml
+```
+Keluarannya akan seperti ini:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  ...
+  name: regcred
+  ...
+data:
+  .dockerconfigjson: eyJodHRwczovL2luZGV4L ... J0QUl6RTIifX0=
+type: kubernetes.io/dockerconfigjson
+```
+
+Nilai dari bidang `.dockerconfigjson` merupakan representasi dalam _base64_ dari data pribadi Docker kamu.
+
+Untuk memahami apa yang ada dalam bidang `.dockerconfigjson`, ubahlah data Secret menjadi format yang bisa terbaca:
+
+```shell
+kubectl get secret regcred --output="jsonpath={.data.\.dockerconfigjson}" | base64 --decode
+```
+
+Keluarannya akan seperti ini:
+
+```json
+{"auths":{"your.private.registry.example.com":{"username":"janedoe","password":"xxxxxxxxxxx","email":"jdoe@example.com","auth":"c3R...zE2"}}}
+```
+
+Untuk memahami apa yang ada dalam bidang `auth`, ubahlah data Secret menjadi format yang bisa terbaca:
+
+```shell
+echo "c3R...zE2" | base64 --decode
+```
+
+Keluarannya, nama pengguna dan kata kunci yang digabungkan dengan `:`, seperti dibawah ini:
+
+```none
+janedoe:xxxxxxxxxxx
+```
+
+Perhatikan bahwa data Secret berisi token otorisasi yang serupa dengan berkas `~/.docker/config.json` lokal kamu.
+
+Kamu telah berhasil menetapkan data pribadi Docker kamu sebagai sebuah Secret yang dipanggil dengan `regcred` pada klaster.
+
+
+## Membuat Pod yang menggunakan Secret kamu
+
+
+Berikut ini adalah berkas konfigurasi untuk Pod yang memerlukan akses ke data pribadi Docker kamu pada `regcred`:
+
+{{< codenew file="pods/private-reg-pod.yaml" >}}
+
+Unduh berkas diatas:
+
+```shell
+wget -O my-private-reg-pod.yaml https://k8s.io/examples/pods/private-reg-pod.yaml
+```
+
+Dalam berkas `my-private-reg-pod.yaml`, ubah `<your-private-image>` dengan tautan ke _image_ dalam register pribadi seperti ini:
+
+```none
+your.private.registry.example.com/janedoe/jdoe-private:v1
+```
+
+Untuk menarik _image_ dari register pribadi, Kubernetes memerlukan data pribadi.
+Bidang `imagePullSecrets` dalam berkas konfigurasi menentukan bahwa Kubernetes harus mendapatkan data pribadi dari Secret yang bernama `regcred`.
+
+Buatlah Pod yang menggunakan Secret kamu, dan verifikasi bahwa Pod tersebut berjalan:
+
+```shell
+kubectl apply -f my-private-reg-pod.yaml
+kubectl get pod private-reg
+```
+
+
+## {{% heading "whatsnext" %}}
+
+
+* Pelajari lebih lanjut tentang [Secret](/docs/concepts/configuration/secret/).
+* Pelajari lebih lanjut tentang [menggunakan register pribadi](id/docs/concepts/containers/images/#menggunakan-registri-privat).
```

Comment:
missing `/`
```suggestion
* Pelajari lebih lanjut tentang [menggunakan register pribadi](/id/docs/concepts/containers/images/#menggunakan-registri-privat).
```

Reply:
FIxed in https://github.com/kubernetes/website/pull/21712/commits/d8c12dd53a0aa2cd007dfc61bbeb2194a5e83cb2